docker-compose.yaml

```diff
@@ -0,0 +1,78 @@
+name: photoview
+
+volumes:
+  mariadb:
+  postgres:
+
+services:
+  mysql:
+    image: mariadb:11.4
+    environment:
+      MYSQL_DATABASE: photoview_test
+      MYSQL_USER: photoview
+      MYSQL_PASSWORD: photosecret
+      MYSQL_RANDOM_ROOT_PASSWORD: yes
+    healthcheck:
+      test:
+      - CMD
+      - mariadb-admin
+      - --user=photoview
+      - --password=photosecret
+      - ping
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+    - mariadb:/var/lib/mysql:rw
+    ports:
+    - 3306:3306
+
+  postgres:
+    image: postgres:16.3
+    environment:
+      POSTGRES_DB: photoview_test
+      POSTGRES_USER: photoview
+      POSTGRES_PASSWORD: photosecret
+    healthcheck:
+      test:
+      - CMD
+      - pg_isready
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    volumes:
+    - postgres:/var/lib/postgresql/data:rw
+    ports:
+    - 5432:5432
+
+  api:
+    image: photoview/photoview-api
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: api
+    env_file: scripts/env.api
+    command: 
+    - /bin/bash
+    - -c
+    - |
+      git config --global --add safe.directory /app
+      source /app/scripts/set_go_env.sh
+      go run .
+    volumes:
+    - .:/app:rw
+
+  ui:
+    image: photoview/photoview-ui
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: ui
+    command:
+    - /bin/bash
+    - -c
+    - |
+      npm ci --omit=dev --ignore-scripts
```

Comment:
Why `-omit=dev` if this would be a dev container?

Reply:
Copied from building Dockerfile. I will remove it.

Reply:
Done.